config/samples/_v1alpha2_orchestrator.yaml

```diff
@@ -0,0 +1,47 @@
+metadata:
+  labels:
+    app.kubernetes.io/name: orchestrator-sample
+  name: orchestrator-sample
+spec:
+  serverlessLogicOperator:
+    enabled: true # Determines whether to install the ServerlessLogic operator. Defaults to True. Optional
+  serverlessOperator:
+    installOperator: true # Determines whether to install the Serverless operator. Defaults to True. Optional
+  rhdh:
+    installOperator: true # Determines whether the RHDH operator should be installed.This determines the deployment of the RHDH instance. Defaults to False. Optional
+    devMode: false # Determines whether to enable the guest provider in RHDH. This should be used for development purposes ONLY and should not be enabled in production. Defaults to False. Optional
+    name: "my-rhdh" # Name of RHDH CR, whether existing or to be installed. Required
+    namespace: "rhdh" # Namespace of RHDH Instance, whether existing or to be installed. Required
+    plugins:
+      notificationsEmail:
+        enabled: false # Determines whether to install the Notifications Email plugin. Requires setting of hostname and credentials in backstage secret. The secret, backstage-backend-auth-secret, is created as a pre-requisite. See value backstage-backend-auth-secret. See plugin configuration at https://github.com/backstage/backstage/blob/master/plugins/notifications-backend-module-email/config.d.ts
+        port: 587 # SMTP server port. Defaults to 587. Optional
+        sender: "" # Email address of the Sender. Defaults to empty string. Optional
+        replyTo: "" # Email address of the Recipient. Defaults to empty string. Optional
+  postgres:
+    name: "sonataflow-psql-postgresql" # The name of the Postgres DB service to be used by platform services. Cannot be empty.
+    namespace: "sonataflow-infra" # The namespace of the Postgres DB service to be used by platform services.
+    authSecret:
+      name: "sonataflow-psql-postgresql" # Name of existing secret to use for PostgreSQL credentials. Required
+      userKey: postgres-username # Name of key in existing secret to use for PostgreSQL credentials. Required
+      passwordKey: postgres-password # Name of key in existing secret to use for PostgreSQL credentials. Required
+    database: sonataflow # Name of existing database instance used by data index and job service. Required
+  platform: # Contains the configuration for the infrastructure services required for the Orchestrator to serve workflows by leveraging the OpenShift Serverless and OpenShift Serverless Logic capabilities.
+    namespace: "sonataflow-infra"
+    resources:
+      requests:
+        memory: "64Mi" # Defines the Memory resource limits. Optional
+        cpu: "250m" # Defines the CPU resource limits. Optional
+      limits:
+        memory: "1Gi" # Defines the Memory resource limits. Optional
+        cpu: "500m" # Defines the CPU resource limits. Optional
+    eventing:
+      name: "my-knative" # Name of existing Broker instance. Optional
```

Comment:
There should be `broker` level after `eventing`, which contains `name` and `namespace`: https://github.com/parodos-dev/orchestrator-helm-operator/pull/247/files#diff-f141cbfd830ae3fdcdc41290144bf39d5bf70abaaa21ba1de22368f0c149e326R82-R95